src/classes/dailyOverdueSummary.cls

```diff
@@ -0,0 +1,33 @@
+global class dailyOverdueSummary implements Schedulable {
+
+	global void execute(SchedulableContext sc) {
```

Comment:
please use the start and finish methods to ensure proper processing

Reply:
@rjulianialtimetrik Schedulable intetrface just have execute(context) method. May be i should use a batch first with the start(), execute() and finish() methods and then call the batch from a schedulable class?